buildings-example/datasets.py

```diff
@@ -1,44 +1,54 @@
 """
 File for storing references to datasets.
 """
+from datetime import date
 from typing import List
 
+import pandas as pd
+
+from openmapflow.config import PROJECT_ROOT, DataPaths
+from openmapflow.constants import END, LAT, LON, START, SUBSET
+from openmapflow.label_utils import train_val_test_split
 from openmapflow.labeled_dataset import LabeledDataset, create_datasets
-from openmapflow.labeled_dataset_custom import CustomLabeledDataset
-from openmapflow.raw_labels import RawLabels
+
+raw_dir = PROJECT_ROOT / DataPaths.RAW_LABELS
+label_col = "is_building"
```

Comment:
A label column helps centralize which column will have the labels but it isn't absolutely necessary.